DeepSpeech.py

```diff
@@ -601,6 +602,8 @@ def calculate_and_print_wer_report(caption, results_tuple):
         item = items[i]
         # If distance > 0 we know that there is a WER > 0 and have to calculate it
         if item[2] > 0:
+            # Replace result by language model corrected result
+            item = (item[0], correction(item[1]), item[2], item[3])
             # Replacing accuracy tuple entry by the WER
             item = items[i] = (item[0], item[1], wer(item[0], item[1]), item[3])
```

Comment:
This line is a bit confusing. Since you're changing this, can you also make this code update `item` and `items` separately? Something like:

```python
# Replacing accuracy tuple entry by the WER
item = (item[0], item[1], wer(item[0], item[1]), item[3])

items[i] = item
```